stable/openebs/values.yaml

```diff
@@ -38,10 +38,15 @@ apiserver:
 defaultStorageConfig:
   enabled: "true"
 
+persistentStoragePath:
```

Comment:
Let us call this - `varDirectoryPath`. The word persistent will get confusing with the actual storage location.  

The comment can say: "This directory is used by the OpenEBS to store debug information and so forth that are generated in the course of running OpenEBS containers."

Reply:
Done